src/settings/mod.rs

```diff
@@ -1,28 +1,72 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, you can obtain one at https://mozilla.org/MPL/2.0/.
-
```

Comment:
Tiny nit, but can we keep this blank line so that the doc comment is visually separated from the license preamble?